### DISCUSSION TITLE
Arquivos png não carregam no github pages

### DISCUSSION THREAD

**Discussion by Gabbaogt** [community/community#60065] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Boa tarde!

Atualmente subi um projetinho meu no Pages, mas percebi que meus arquivos png não são carregados, mesmo com o código certo não carrega, só carrega os arquivos JPEG.

Alguém poderia me ajudar a solucionar essa problema?

Grato a todos

**Comment by GhostOf0days** [comment, marked as answer]
Sorry for answering this question in English, but the problem lies in how you are defining the paths. You are using relative pathing for referencing the images. index.html as at the root. The images folder is one layer deeper. ./ states that you want to start at the directory that the file you are on is (index.html). ../ states that you want to start one directory back from the file you are on.

So let's take one of the src attributes you define: ../imagens/logogithub100.png.

This states that the file structure looks like this.:

1 layer deep: index.html
at root: imeagens
1 layer deep inside imagens: logogithub100.png

However, the correct structure is this:
at root: index.html
1 layer deep: imeagens
1 layer deep inside imagens: logogithub100.png

tl;dr: The way to fix this is changing all of your src attribute paths for images from ../imagens/imageName to ./imagens/imageName. Essentially, remove one dot at the beginning.
